scripts/output_env.sh

```diff
@@ -94,6 +99,15 @@ echo
 print_version "gdb" "--version" "" "head -n 1"
 echo
 
+print_version "perl" "--version" "" "head -n 2" "grep ."
+echo
+
+print_version "python" "--version" "" "head -n 1"
+echo
+
+print_version "pylint3" "--version" "" "head -n 2" "tail -n 1"
```

Comment:
On the CI machines, this gives
`pylint3: /usr/bin/pylint3 : astroid 1.4.4`
On my own machine, I get 
```
No config file found, using default configuration
astroid 1.4.4
```

Reply:
On my machine, it gives
``` * pylint3: /usr/bin/pylint3 : pylint3 1.8.3,  ```

Reply:
To help with fixing this, my output of `pylint3 --version` is
```
No config file found, using default configuration
pylint3 1.5.2, 
astroid 1.4.4
Python 3.5.2 (default, Oct  8 2019, 13:06:37) 
[GCC 5.4.0 20160609]
```

Reply:
Maybe use `grep pylint3` to get just the line with the pylint3 version?

Reply:
On my end it is:
```$ pylint3 --version
Using config file ~/Desktop/Workspace/mbedtls/.pylintrc
pylint3 1.8.3, 
astroid 1.6.0
Python 3.6.8 (default, Oct  7 2019, 12:59:55) 
[GCC 8.3.0]
```

Reply:
Is there no pylint version in there at all?

Reply:
I think I've worked out whats happening. It looks like older versions of pylint3 don't specify what config file they're using if one exists. This means the pylint3 version is on different lines depending on what version it is.